Revit_Core_Engine/Query/Profile.cs

```diff
@@ -79,10 +79,30 @@ public static IProfile Profile(this Autodesk.Revit.DB.Mechanical.Duct duct, Revi
             }
         }
 
+        /***************************************************/
+        
+        [Description("Extract a BHoM cable tray profile from a Revit cable tray.")]
+        [Input("cableTray", "Revit cable tray to extract relevant profile information from.")]
+        [Input("settings", "Revit adapter settings.")]
+        [Output("profile", "BHoM cable tray profile extracted from a Revit cable tray.")]
+        public static IProfile Profile(this Autodesk.Revit.DB.Electrical.CableTray cableTray, RevitSettings settings = null)
+        {
+            settings = settings.DefaultIfNull();
+            
+            // Cable Trays are always rectangular
+            double boxHeight = cableTray.Height.ToSI(UnitType.UT_Electrical_CableTraySize);
+            double boxWidth = cableTray.Width.ToSI(UnitType.UT_Electrical_CableTraySize);
+            double boxThickness = 0.001519;
```

Comment:
Is this the appropriate thickness for a cable tray? This assumption is a carry over from duct gauge thickness default being associated with 16-gauge duct

Reply:
Yep carried it over from ducts, have you got a suggestion for a value here?